components/net_traits/bluetooth_scanfilter.rs

```diff
@@ -54,12 +60,14 @@ impl BluetoothScanfilterSequence {
     pub fn new(vec: Vec<BluetoothScanfilter>) -> BluetoothScanfilterSequence {
         BluetoothScanfilterSequence(vec)
     }
-}
 
-impl BluetoothScanfilterSequence {
-    fn has_empty_or_invalid_filter(&self) -> bool {
-        self.0.is_empty() &&
-        self.0.iter().all(|x| !(x.is_empty_or_invalid_filter()))
+    pub fn has_empty_or_invalid_filter(&self) -> bool {
```

Comment:
Maybe we can stay by this name, because if we found an empty or invalid filter the iteration will stop.